### PR TITLE
Code loading page #1048

### PR DESCRIPTION
**Pull request by davidwiese**
Fixes issue #1048 

Created the spinner in pure CSS and implemented the `<Loading />` component directly in `index.tsx` for speed. Tested in Firefox and Chrome, mobile and desktop responsive. Feels snappy.

mobile

desktop

### CODE REVIEW

File: products/statement-generator/src/index.tsx [hunk 1]
```diff
@@ -11,13 +12,21 @@ import './i18n';
...
   <React.StrictMode>
-    <Suspense fallback={<div>Loading...</div>}>
+    <Suspense fallback={<Loading />}>
       <App />
     </Suspense>
   </React.StrictMode>,
   document.getElementById('root')
 );
 
+function Loading() {
```
Comment (sydneywalcoff) [inline products/statement-generator/src/index.tsx:22]: @davidwiese, lets make this its own component. I think it makes sense to put it in the `/components-layout`, but flexible to other ideas.

File: products/statement-generator/src/styles/index.module.css
Comment (sydneywalcoff) [inline products/statement-generator/src/styles/index.module.css:1]: When you separate the Loading component, can you translate this into Material UI please? It looks like they may actually have a [component](https://mui.com/material-ui/react-progress/) that would make your life much easier.

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
Hey @davidwiese! Looking great so far!

I wrote an inline comments, but I'll add them here too.
- Can you separate the loading component into its own file and store under the components-layout folder? I'm flexible to storing it elsewhere if you have other ideas though. 
- Can we translate that css file into Material UI please? It looks like they may actually have a [component](https://mui.com/material-ui/react-progress/) that would make your life much easier.

File: products/statement-generator/src/index.tsx (hunk 1 above)
Comment (davidwiese) [reply products/statement-generator/src/index.tsx:22]: Done!

File: products/statement-generator/src/styles/index.module.css
Comment (davidwiese) [reply products/statement-generator/src/styles/index.module.css:1]: Done!

**Comment by davidwiese** [comment]
Made separate `Loading` component and used the Material UI Circular Progress component

File: products/statement-generator/src/components-layout/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CircularProgress } from '@material-ui/core';
+import styles from '../styles/Loading.module.css';
+
+interface LoadingProps {}
+
+const Loading: React.FC<LoadingProps> = () => {
+  return (
+    <div className={styles.wrapper}>
```
Comment (sydneywalcoff) [inline products/statement-generator/src/components-layout/Loading.tsx:9]: @davidwiese can we translate this to Material UI classes also? You can make custom classes that you apply similarly to how you did here. Check out other components to see how the Material UI custom classes work.

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
@davidwiese one more thing! Ideally we want to do all our styling through Material UI before we add an external stylesheet. Check my comment below

File: products/statement-generator/src/components-layout/Loading.tsx (hunk 2 above)
Comment (davidwiese) [reply products/statement-generator/src/components-layout/Loading.tsx:9]: Hadn't done this before, good learning experience

**Review by sydneywalcoff — APPROVED** [review]
@davidwiese Thanks for making those changes! Looks great!! 🎉

**Pull request merged**